mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+site_name: RESTier Documentation
+site_description: How to use the RESTier framework for .NET.
+theme: readthedocs
+pages:
+- Home:
+    - 'Introduction': 'index.md'
+    - 'Getting Started': 'getting-started.md'
```

Comment:
How you think about section header like 
1. Introduction
2. Getting Started
3. Building the Services
3.1 Entity Set Filters